care/facility/models/patient_consultation.py

```diff
@@ -26,9 +27,24 @@
     reverse_choices,
 )
 from care.users.models import User
+from care.utils.models.base import BaseModel
 from care.utils.models.validators import JSONFieldSchemaValidator
 
 
+class ConsultationSymptoms(BaseModel, ConsultationRelatedPermissionMixin):
```

Comment:
```suggestion
class ConsultationSymptom(BaseModel, ConsultationRelatedPermissionMixin):
```

Model name should be singular